hackyballs: Use common widget for popup choices menu

ptomato opened the pull request:
This uses the common popup menu widget that we also use in the framework
toolbox, for consistency. The common widget visually selects the chosen
option, which would otherwise need to be solved separately in the
hackyballs toolbox code.

https://phabricator.endlessm.com/T23984

ptomato committed:
hackyballs: Use common widget for popup choices menu

This uses the common popup menu widget that we also use in the framework
toolbox, for consistency. The common widget visually selects the chosen
option, which would otherwise need to be solved separately in the
hackyballs toolbox code.

https://phabricator.endlessm.com/T23984

tchx84 commented:
@ptomato thanks for taking over this one. One comment:

It's not syncing with the initial state. e.g. open the app and the toolbar (for the first time), go to the "Balls 1" tab (the second one), and you will see that the button is not displaying the right icon.

I also see this error (probably related):
```
"JS ERROR: Gio.ResourceError: The resource at “/com/endlessm/HackToolbox/hackyballs/skins/undefined.png” does not exist"
```

tchx84 reviewed (changes requested):
One small regression.

ptomato committed:
hackyballs: Make sure pixels property is defined

This gets rid of a warning when creating a new image button.

https://phabricator.endlessm.com/T23984
